integreat_cms/cms/forms/media/replace_media_file_form.py

```diff
@@ -130,10 +130,12 @@ def clean(self) -> dict:
         return cleaned_data
 
     def save(self, commit: bool = True) -> MediaFile:
-        # Remove old file
-        os.remove(self.original_file_path)
-        logger.debug("Removed old file %r", self.original_file_path)
-
+        # Remove old
+        try:
+            os.remove(self.original_file_path)
+            logger.debug("Removed old file %r", self.original_file_path)
+        except FileNotFoundError:
+            logger.debug("The file %r could not be removed", self.original_file_path)
```

Comment:
The test file is not stroed during the test and it caused an error as the path does not exists. So `try` and `except` are added. I hope this does not harm the functionality and bring some safety in practice.